app/controllers/articles_controller.rb

```diff
@@ -1,6 +1,10 @@
 class ArticlesController < ApplicationController
-  def index
-    @articles = Article.all
+
+  def index    
+    @user = helpers.current_user
```

Comment:
@JuanMaGiraldo why `helpers.current_user` is required?

Reply:
In this case, to obtain the currently logged-in user, this in order to obtain the articles written by him. This function is defined in ApplicationHelper so the controllers can also use it, in this way avoid repeating code.

Reply:
@JuanMaGiraldo I think a better approach is to have these helper methods (under helpers.<>) placed in the ApplicationController with it you will ensure method access from the controllers, views and templates without having to specify the `helpers.` what do you think?

Reply:
@JuanMaGiraldo please correct it and push your commits to this branch!